File: DAM_1er_año/Programacion/Java/Repaso/src/Ej5.java
import java.util.Scanner;

public class Ej5 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Dime un número para cuenta regresiva: ");
        int numero = scanner.nextInt();
        for (int i = numero; i > 0; i--) {
            System.out.println(i);
        }
        System.out.println("Acabó la cuenta regresiva");
        scanner.close();
    }
}

File: DAM_1/DAM_1er_año/Programacion/Python/Ej_Recursividad/Ej5.py
def es_primo(n,d=2):
    if n <= 1:
        return False
    elif n == 2:
        return True
    elif n % d == 0:
        return False
    elif d * d > n:
        return True
    return es_primo(n, d + 1)


n1 = int(input("Dime tu número: "))

resultado = es_primo(n1)
print("Tu número es:" , resultado)